File: app/src/main/java/com/organizobra_mobile/ui/people/PeopleViewModel.java
package com.organizobra_mobile.ui.people;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class PeopleViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public PeopleViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is people fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/organizobra_mobile/ui/people/PeopleViewModel.kt
package com.organizobra_mobile.ui.people

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class PeopleViewModel : ViewModel() {
    private val mText = MutableLiveData<String>()

    init {
        mText.value = "This is people fragment"
    }

    val text: LiveData<String>
        get() = mText
}